### ISSUE TITLE
Verification problem before logging in

### ISSUE THREAD

**Issue by anting-shu** [mengxu98/thaiticketmajor-script#1]
There is no way to verify it.

**Comment by mengxu98** [comment]
Yes, the current version of Python code cannot solve this problem, and we may try to further optimize it in the future. I suggest you try using the JS version of the code. If you still have problems, please feel free to submit an issue.
